### ISSUE TITLE
When saving Asset Bundle files at larger file sizes, it exports corrupted file (even with no changes in bytes to bundle contents)

### ISSUE THREAD

**Issue by BowDown097** [DerPopo/UABE#239]
So with a game called Bloons TD 6, I'm attempting to mod an asset bundle file which is ~250 MB. All goes fine, but when I export it after a mod, the file gets roughly 20 KB larger and therefore I cannot run the game with the modded file as it views the file as corrupt. Even without changing the byte size of the bundle contents at all, this issue still happens.
Although, if I do this with an asset bundle file that's around 100 MB it will export fine with no added KB, and the game reads it perfectly fine.

**Comment by AsukiKanetaka** [comment]
This also happens to me with one of their games.

**Comment by DerPopo** [comment]
The files related to that bug (or at least a similar one) which I have seen don't seem to be totally corrupted. I think there is some smaller issue related to data alignment or in which order the assets are stored that Unity doesn't like.

**Comment by BowDown097** [comment]
If you want, I can provide the asset bundle in question.

**Comment by BowDown097** [comment]
Also, it's either an issue with bundle version or bundle size. I used an older version of a "resource bundle" from dev builds of the game from before it came out, and the bundles for this build in particular used Unity 2018.1 and the resource bundle was also < 200 MB and I was able to modify it perfectly fine. When the game came out, they began to use Unity 2018.2 for bundles and the resource bundle was > 200 MB and I couldn't modify it.
So idk
